checkstyle.properties

```diff
@@ -1,12 +1,24 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16
-publicVersions=4.17
+archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16,4.17,4.18
+publicVersions=4.19
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+4.19.description=Upgrade to Checkstyle 8.19
+4.19.sqVersion=[6.7,7.7+]
```

Comment:
This range should end with `LATEST`. As-is, the plugin will be marked incompatible with the imminent 7.8 release.

Reply:
Thank you! :)
I have changed it to `LATEST`.